### PR TITLE
add hindi translation for remote-schemas.md

### PR DESCRIPTION
**Pull request by anchal7299**
I translated remote-schemas file in hindi.

### CODE REVIEW

**Comment by anchal7299** [comment]
What will be the url for Hindi translation?

**Comment by marionschleifer** [comment]
@anchal7299 you can just add a file in the /translations folder mentioned above. Meanwhile, could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Comment by anchal7299** [comment]
Ohk
Do I need to send my address on this email?

**Comment by marionschleifer** [comment]
@anchal7299 just send an email with the link to this PR, and I'll send you a form to fill in 🙂

**Comment by marionschleifer** [comment]
@anchal7299 I'm closing this PR. Feel free to re-open if you would like to implement the suggested changes 🙂

**Comment by anchal7299** [comment]
@marion, since my end Semester exams are going on that's why i was unable
to make the suggested changes. Please reopen the PR. I will submit the
changes by tomorrow.

File: translations/remote-schemas.hindi.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+अगर आप एक ही जगह से सभी प्रकार की GraphQL की क्वेरी करना चाहते हैं तो रिमोट GraphQL स्कीमा को GraphQL इंजन के Postgres आधारित स्कीमा के साथ विलय करे। रिमोट स्कीमा कुछ उदाहरण के लिए आदर्श होते हैं जैसे:
+
+* म्यूटेशन का अनुकूलन (जैसे इन्सर्ट से पहले वेलिडेशन करना)
+* भुगतान जैसी सुविधाओं का समर्थन करना, आदि और उन्हें एक्सेस करने के लिए एक सुसंगत इंटरफ़ेस प्रदान करना यानि GraphQL Engine के API के पीछे
+* अन्य सूत्रों से डेटा अलग करना (*जैसे मौसम के एपीआई या किसी अन्य डेटाबेस से*)
+
+कस्टम व्यवसाय तर्क का समर्थन करने के लिए, आपको एक कस्टम GraphQL सर्वर बनाने की आवश्यकता होगी (देखे [boilerplates](community/boilerplates/remote-schemas)) और उसके स्कीमा को GraphQL Engine के साथ विलय करे।
+
+![remote schems architecture](assets/remote-schemas-arch.png)
```
Comment (marionschleifer) [inline translations/remote-schemas.hindi.md]: Link doesn't work and diagram doesn't show up:

image

File: translations/remote-schemas.hindi.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+   }
+   ```
+
+   आप ``GraphiQL`` इंटरफ़ेस के शीर्ष दाएं कोने में ``Docs`` एक्सप्लोरर का उपयोग करके रिमोट स्कीमा से GraphQL के प्रकारों का पता लगा सकते हैं।
+
+## Boilerplates
+
+लोकप्रिय भाषाओं / फ़्रेमवर्क्स में कस्टम GraphQL सर्वर के लिए Boilerplates उपलब्ध हैं।
+
+* [नियमित boilerplates](community/boilerplates/graphql-servers) जिसे कहीं भी deploy किया जा सकता है।
```
Comment (marionschleifer) [inline translations/remote-schemas.hindi.md]: Link doesn't work.

File: translations/remote-schemas.hindi.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+## प्रलेखन
+
+पूरा पढ़ें [प्रलेखन](https://docs.hasura.io/1.0/graphql/manual/remote-schemas/index.html).
+
+## अनुवाद
+
+यह दस्तावेज़ निम्नलिखित अनुवादों में उपलब्ध है:
+
+- [French :fr:](translations/remote-schemas.french.md)
+- [Hindi :in:](translations/remote-schemas.hindi.md)
```
Comment (marionschleifer) [inline translations/remote-schemas.hindi.md]: Link doesn't work and flag doesn't show up:

image

File: remote-schemas.md [hunk 4]
```diff
@@ -81,3 +87,4 @@ Read the complete [documentation](https://docs.hasura.io/1.0/graphql/manual/remo
 This document is available in the following translations:
 
 - [French :fr:](translations/remote-schemas.french.md)
+- [Hindi ](translations/remote-schemas.hindi.md)
```
[the remark this answers is not in the document]
Comment (anchal7299) [reply remote-schemas.md]: I tried to add flag here but it's not showing. Can you help me in this issue?

File: translations/remote-schemas.hindi.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+   }
+   ```
+
+   आप ``GraphiQL`` इंटरफ़ेस के शीर्ष दाएं कोने में ``Docs`` एक्सप्लोरर का उपयोग करके रिमोट स्कीमा से GraphQL के प्रकारों का पता लगा सकते हैं।
+
+## Boilerplates
+
+लोकप्रिय भाषाओं / फ़्रेमवर्क्स में कस्टम GraphQL सर्वर के लिए Boilerplates उपलब्ध हैं।
+
+* [नियमित boilerplates](../community/boilerplates/graphql-servers) जिसे कहीं भी deploy किया जा सकता है।
```
Comment (marionschleifer) [inline translations/remote-schemas.hindi.md]: This link is still not working. Try `../community/boilerplates`

File: translations/remote-schemas.hindi.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+## प्रलेखन
+
+पूरा पढ़ें [प्रलेखन](https://docs.hasura.io/1.0/graphql/manual/remote-schemas/index.html).
+
+## अनुवाद
+
+यह दस्तावेज़ निम्नलिखित अनुवादों में उपलब्ध है:
+
+- [French :fr:](translations/remote-schemas.french.md)
+- [Hindi ](translations/remote-schemas.hindi.md)
```
Comment (marionschleifer) [inline translations/remote-schemas.hindi.md]: This is not needed as we're already in the Hindi translation.

File: remote-schemas.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (marionschleifer) [reply remote-schemas.md]: Try copy-pasting this flag: 🇮🇳

**Pull request merged**